query-engine/schema/src/query_schema.rs

```diff
@@ -105,7 +105,8 @@ impl QuerySchema {
     }
 
     pub(crate) fn can_full_text_search(&self) -> bool {
-        self.has_feature(PreviewFeature::FullTextSearch) && self.has_capability(ConnectorCapability::FullTextSearch)
+        // TODO: add connector-specific `self.has_native_feature(ConnectorCapability::NativeFullTextSearch)` in conjunction with the next bool.
```

Comment:
I think this is still missing in the PR, right now this method will return `true` for Postgres without the preview feature